src/routes/landing/containers/LandingContainer.js

```diff
@@ -67,7 +67,7 @@ class LandingContainer extends React.Component<ContainerPropsType> {
     })
 
     navigation.navigate({
-      routeName: 'App',
+      routeName: 'CityContent',
```

Comment:
:+1: Did you test the navigation after the refactor (Modals, Nagivation back etc.)?

Reply:
Yup, tested it.